examples/disaster-recovery/README.md

```diff
@@ -0,0 +1,49 @@
+# Coordinate disaster recovery with Secret Manager
+
+## Background
+
+This document assumes you are already using the following strategy for
+detecting and triggering failovers:
+1. Using an independent service to detect when the primary is down
+2. Trigger a promotion of an existing read replica to become a primary
+3. Update a Secret Manager secret with the name of the current primary
+
+## Restart Auth proxy when secret changes
+
+This option uses a wrapper script around the Cloud SQL Auth proxy to detect
+when the secret has changed, and restart the proxy with the new value. This
+could be done in many languages, but here’s an example using bash:
+
+```sh
```

Comment:
Can we include a link to the file that contains this snippet? e.g. :https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/2a9c8d8cb24e2c84e43620dab333677191d1dbd7/examples/k8s-sidecar/README.md?plain=1#L24

Reply:
good call 👍